x-pack/plugins/infra/public/containers/logs/log_filter/log_filter_state.ts

```diff
@@ -7,7 +7,7 @@
 
 import createContainer from 'constate';
 import { useCallback, useState } from 'react';
-import { useDebounce } from 'react-use';
+import useDebounce from 'react-use/lib/useDebounce';
```

Comment:
This PR fixes an eslint rule that should have been firing, but wasn't. I've updated this import to comply with the fixed rule.